spec/overcommit/hook/pre_commit/rake_target_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe Overcommit::Hook::PreCommit::RakeTarget do
+  let(:config)  { Overcommit::ConfigurationLoader.default_configuration }
+  let(:context) { double('context') }
+  subject { described_class.new(config, context) }
+
+  # context 'without targets parameters' do
+  #   let(:result) { double('result') }
+  #   it 'raises' do
+  #     expect { subject }.to raise_error(RuntimeError)
```

Comment:
but if testing with a real example:

```
Hook raised unexpected error
RakeTarget: targets parameter is empty. Add at least one task to           the targets parameter. Valid: Array of target names or String of           target names
```

and it seems like i need to break lines in another way in the code.